src/App.tsx

```diff
@@ -1,61 +1,119 @@
 import './App.scss';
+import { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { User } from './components/types/User';
+import { Todo } from './components/types/Todo';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+function findUser(todoId: number): User | null {
+  const user = usersFromServer.find(person => person.id === todoId);
+
+  return user || null;
+}
+
+const todosWithUser: Todo[] = todosFromServer.map(todo => (
+  {
+    ...todo,
+    user: findUser(todo.userId),
+  }
+));
 
 export const App = () => {
+  const [title, setNewTitle] = useState('');
+  const [userId, setNewId] = useState(0);
+  const [userSelected, isUserSelected] = useState(true);
+  const [titleCreated, isTitleCreated] = useState(true);
+
+  const maxTodoId = Math.max(...todosWithUser.map(todo => todo.id)) + 1;
+
+  const addNewTodo = (todoId: number) => {
+    const newUser = usersFromServer.find(user => user.id === todoId);
```

Comment:
i don't understand why todoId === user.id ? And why it's named as newUser? if it is already exist?

Reply:
what is common between user and todo id?

Reply:
![todoAdd-form-userInfo](https://user-images.githubusercontent.com/112826621/228034153-3aaa5a64-0cc2-4831-a8d1-c8f9661a080b.jpg)
I am trying to find a user who created a new to do ( as fas as I get it, todo`s userId and user`s id are the same numbers, actually todoId is userId cause it is passed as an argument in addNewTodo function). Then I use found user as a value for key 'user' in each new todo. Smth like this. As for the name 'newUser', would it be better if i rename it to 'newTodocreator' for example to make it more explicit?